Dockerfile

```diff
@@ -60,7 +60,7 @@ RUN make -j$(nproc)
 
 
 # Build the dockerpi VM image
-FROM busybox:1.31 AS dockerpi-vm
+FROM busybox:1.34 AS dockerpi-vm
```

Comment:
Latest version is 1.36 per https://hub.docker.com/_/busybox/tags

Reply:
Thanks @rasa! I'd be happy to update the PR if needed, but doesn't look like this project is being maintained :(

Hopefully having these PRs open can still help people encountering this issue.

I've been using a simple fork with this patch for my own images based on top of this one: https://github.com/carlosperate/dockerpi/pkgs/container/dockerpi/12649266?tag=vm-fix